lms/templates/student_account/login_and_register.html

```diff
@@ -22,6 +22,7 @@
             newrelic.addPageAction('xfinished');
         }
     </%static:require_module>
+    <script type="text/javascript" src="${static.url('js/student_account/tos_modal.js')}"></script>
```

Comment:
Thanks @douglashall. I implemented it using your suggestion. One thing I noticed though is that for the site configuration setting to take effect I need to restart my devstack/sandbox. Is that normal or maybe I haven't set my site correctly?